code/game/objects/structures/crates_lockers/closets/job_closets.dm

```diff
@@ -40,7 +39,6 @@
 
 /obj/structure/closet/chefcloset/WillContain()
 	return list(
-		/obj/item/clothing/under/sundress,
```

Comment:
i dont see whats wrong with this one, service staff dressing nicely isnt too bad